pkgs/by-name/ne/nexus-mods-app/package.nix

```
@@ -0,0 +1,125 @@
+{ _7zz
+, buildDotnetModule
+, copyDesktopItems
+, desktop-file-utils
+, dotnetCorePackages
+, fetchFromGitHub
+, fontconfig
+, lib
+, libICE
+, libSM
+, libX11
+, nexus-mods-app
+, runCommand
+, enableUnfree ? false # Set to true to support RAR format mods
+}:
+let
+  pname = "nexus-mods-app";
+  version = "0.4.1";
+  src = fetchFromGitHub {
+    owner = "Nexus-Mods";
+    repo = "NexusMods.App";
+    rev = "v${version}";
+    fetchSubmodules = true;
+    hash = "sha256-FzQphMhiC1g+6qmk/R1v4rq2ldy35NcaWm0RR1UlwLA=";
+  };
+  meta = with lib; {
```

```suggestion
  meta = {
```

tracking issue: #292468

---

I agree, and I do that for all of my own code, but I've been told to do the opposite before. And the tracking issue doesn't seem to be resolved in any particular way.

---

There's no technical argument to keep it, only an aesthetic one. And there's a technical argument to remove it: https://github.com/NixOS/nixpkgs/issues/292468#issuecomment-2016454242

---

Done, thanks for the review!